source/_components/sensor.teleinfo.markdown

```diff
@@ -0,0 +1,81 @@
+---
+layout: page
+title: "EDF Teleinfo"
+description: "Instructions on how to integrate an EDF teleinfo sensor into Home Assistant."
+date: 2017-11-15 19:08
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: edf.png
+ha_category: Sensor
+ha_iot_class: "Energy"
+ha_release: "0.61"
+---
+
+The `teleinfo` sensor will read data stream from french electric meters. (Description in french
+[here](http://www.enedis.fr/sites/default/files/Enedis-NOI-CPT_02E.pdf). 
+
+To enable this sensor, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+
+- platform: teleinfo
+  name: "edf"
+  device: "/dev/ttyUSB0"
+
+- platform: template
+  sensors:
+    teleinfo_hchc:
+      value_template: '{{ states.sensor.edf.attributes["HCHC"] }}'
```

Comment:
These should be using `"{{ state_attr('sensor.edf','HCHC') }}"` as this format handles if the sensor isn't available, rather than throwing errors.